csrc/scheduler/pointwise_utils.h

```diff
@@ -127,5 +127,20 @@ std::optional<CommonScheduleInfo> commonPointwiseSchedule(
     Fusion* fusion,
     int64_t break_point);
 
+// Compute vectorization factor with special handling for sub-byte data types
```

Comment:
We also have `vectorize_helper::getVectorizationFactor`. I understand that this function has one more additional thing to do, but the naming seems confusing. I wouldn't immediately know which to use given `computeVectorizationFactor` and `getVectorizationFactor`.

IIUC, this sub-byte constraint is not specific to a particular scheduler. Why not merge it into `getVectorizationFactor`?